modules/replicator/src/integrationTest/java/org/apache/ignite/internal/replicator/ItPlacementDriverReplicaSideTest.java

```diff
@@ -157,7 +158,7 @@ public void beforeTest(TestInfo testInfo) {
                     cmgManager,
                     clock,
                     Set.of(ReplicaMessageTestGroup.class),
-                    new TestPlacementDriver(nodeName)
+                    new TestPlacementDriver(clusterService)
```

Comment:
As was mentioned above, because we only need nodeName and nodeId inside, let's use them instead of the service that provides mass of varieties.

Reply:
Answered above.